x-pack/test/api_integration/config.js

```diff
@@ -47,3 +46,5 @@ export default async function ({ readConfigFile }) {
     esTestCluster: xPackFunctionalTestsConfig.get('esTestCluster'),
   };
 }
+
+export default getApiIntegrationConfig;
```

Comment:
Just a little trick I saw in another test file to get my test config to "extend" this one in an easier way.